src/app/common/components/IsValid/IsValid.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { observer } from 'mobx-react';
+import Grid from '@material-ui/core/Grid';
+import { Tooltip } from '@material-ui/core';
+import { IsValidProps, useStyles } from './IsValid.styles';
+
+const IsValid = (props: IsValidProps) => {
```

Comment:
1. `IsValid` name is too common. If it's not going to be used for both validators and operators - then the name should be `IsValidValidatorBadge`, otherwise it should be `IsValidBadge`.
2. If the component is intended to be common - then the title should be passed through the props instead of hardcoded in a component.
3. `IsValidProps` is related only to component itself, so it should be in the same file where component is located, not in his styles and should be renamed to `IsValidBadgeProps`

Final component usage should look something like this, if we talk about keeping this component common:

```
<IsValidBadge
  title="Validator contains malformed public key..."
  isValid={validator.is_valid}
/>
```